### DISCUSSION TITLE
Multi Climate Entities Offline

### DISCUSSION THREAD

**Discussion by luke1699** [Blackymas/NSPanel_HA_Blueprint#2093] [Ideas]
Hi, the is a very good project.
I’d like know if it’s possibile to have two Or more Climate entities offline, when HA is offline.
I’d like control 8 Relay connect to the NSPanel, with 8 different temperature sensors.
Thanks.

**Comment by edwardtfn** [comment]
Are you going to modify your NSPanel to phisically add 6 relays and temperature sensors?

**Reply by luke1699** [reply]
Yes, i want modify it with a custom PCB.

I'd like know how the structure of the code will be it in that case.

**Reply by luke1699** [reply]
There is a possibility to do this?
